__tests__/app.test.js

```diff
@@ -109,6 +109,57 @@ describe("get article by id", () => {
       });
   });
 });
+
+describe("get comments by article id", () => {
+  it("200: most recent comments first", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comments = body.comments;
+        expect(comments).toBeSorted({
+          descending: true,
+          key: "created_at",
+        });
+      });
+  });
+  it("404: returns message if path not found", () => {
+    return request(app)
+      .get("/api/articles/404044/comments")
```

Comment:
just reconsider the error `text` on this error - the error here is that the article hasn't been found in the database - Path not found relates to a bad path i.e. `/api/invalid-endpoint/` which is something we've already tested for

Reply:
Again I agree that the message being sent back here is not correct. This test should be looking more like:
``` 
expect(msg).toBe("Article not found")
```

Reply:
I think your selectArticleById needs amending as well as tests relating to this 404 article not found sad path